corehq/apps/cloudcare/static/cloudcare/js/formplayer/spec/split_screen_case_search_spec.js

```diff
@@ -0,0 +1,123 @@
+/* eslint-env mocha sinon */
+/* global Backbone, Marionette */
+
+describe('Split Screen Case Search', function () {
+    const API = hqImport("cloudcare/js/formplayer/menus/api"),
+        Controller = hqImport('cloudcare/js/formplayer/menus/controller'),
+        FakeFormplayer = hqImport('cloudcare/js/formplayer/spec/fake_formplayer'),
+        FormplayerFrontend = hqImport('cloudcare/js/formplayer/app'),
+        splitScreenCaseListResponse = hqImport('cloudcare/js/formplayer/spec/fixtures/split_screen_case_list'),
+        Toggles = hqImport('hqwebapp/js/toggles'),
+        Utils = hqImport('cloudcare/js/formplayer/utils/utils');
+
+    const currentUrl = new Utils.CloudcareUrl({appId: 'abc123'}),
+        sandbox = sinon.sandbox.create(),
+        stubs = {},
+        REGIONS = {
+            main: 'main',
+            sidebar: 'sidebar',
+        };
+
+    let getRegion;
+
+    before(function () {
+        sandbox.stub(Marionette.CollectionView.prototype, 'render').returns();
+        sandbox.stub(Utils, 'currentUrlToObject').callsFake(function () {
+            return currentUrl;
+        });
+
+        sandbox.stub(Backbone.history, 'start').callsFake(sandbox.spy());
+        sandbox.stub(Backbone.history, 'getFragment').callsFake(function () {
+            return JSON.stringify(currentUrl);
+        });
+        sandbox.stub(API, 'queryFormplayer').callsFake(FakeFormplayer.queryFormplayer);
+
+        stubs.show = sandbox.stub().callsFake(function () { return; });
+        stubs.empty = sandbox.stub().callsFake(function () { return; });
+        FormplayerFrontend.regions = {
+            getRegion: function (region) {
+                return {
+                    region: region,
+                    show: stubs.show,
+                    empty: stubs.empty,
+                };
+            },
+            addRegions: function () { return; },
+        };
+        getRegion = sandbox.spy(FormplayerFrontend.regions, 'getRegion');
+        stubs.sidebarEnabled = sandbox.stub(Toggles, 'toggleEnabled').withArgs('SPLIT_SCREEN_CASE_SEARCH');
+    });
+
+    beforeEach(function () {
+        FormplayerFrontend.currentUser.displayOptions.singleAppMode = false;
+        stubs.sidebarEnabled.returns(true);
+    });
+
+    afterEach(function () {
+        sandbox.resetHistory();
+    });
+
+    after(function () {
+        sandbox.restore();
+    });
+
+    it('should show sidebar and main regions when using split screen case search', function () {
+        Controller.showMenu(splitScreenCaseListResponse);
+
+        assert.isTrue(getRegion.calledWith(REGIONS.sidebar));
+        assert.isTrue(_.some(stubs.show.getCalls(), call => call.thisValue.region === REGIONS.sidebar));
+
+        assert.isTrue(getRegion.calledWith(REGIONS.main));
+        assert.isTrue(_.some(stubs.show.getCalls(), call => call.thisValue.region === REGIONS.main));
+    });
+
+    it('should show sidebar and main regions even when response type query', function () {
```

Comment:
this would be true for response type entities as well

Reply:
The `fakeFormplayer` response being used here has response type entities by default, so I think that's already covered. Is there a situation I'm not considering and/or is it worth clarifying that it's (almost) always an `entities` response?